e2e/tests/ui/layout/search/grandsearch.e2e.spec.js

```diff
@@ -0,0 +1,112 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+/*
+This test suite is dedicated to tests which verify search functionality.
+*/
+
+const { test, expect } = require('@playwright/test');
```

Comment:
This test looks great.

Just a few things:
1. We should keep the functional assertions in their own test separate from the percy tests. The percy client consumes local env variables and so we're keeping those tests in their own folder (./e2e/tests/visual) so that they cannot get executed when we copy these tests to VISTA/ VIPER / Anywhere private. We can put it in the existing default.visual.spec.js file.
2. With the functional test, we should include the fixtures.js file to extend the test class to include console error checks.

Reply:
Done!